robot-simulator/robot_simulator_tests.erl

```diff
@@ -0,0 +1,96 @@
+-module( robot_simulator_tests ).
+-include_lib( "eunit/include/eunit.hrl" ).
+
+create_test() ->
+	       Robot = robot_simulator:create(),
+	       ?assert( robot_simulator:direction(Robot) =:= undefined ),
+	       ?assert(	robot_simulator:position(Robot) =:= {undefined, undefined} ).
+
+place_test() ->
+            Robot = robot_simulator:create(),
+	    Position = {1, 2},
+	    robot_simulator:place( Robot, north, Position ),
+	    ?assert( robot_simulator:direction(Robot) =:= north ),
+	    ?assert( robot_simulator:position(Robot) =:= Position ).
```

Comment:
And if you can be bothered, don't feel obliged, @kytrinyx lining up the tests would be nice too. /nitpick